embedded-graphics/src/geometry/angle.rs

```diff
@@ -0,0 +1,477 @@
+use super::real;
+use crate::geometry::Real;
+use core::f32::consts::PI;
+use core::ops::{Add, AddAssign, Neg, Sub, SubAssign};
+use float_cmp::{ApproxEq, F32Margin};
+#[cfg(not(feature = "fixed_point"))]
+#[allow(unused_imports)]
+use micromath::F32Ext;
+
+pub(crate) mod angle_consts {
+    use super::{real, Angle};
+
+    pub(crate) const ANGLE_90DEG: Angle = Angle(real::FRAC_PI_2);
+    pub(crate) const ANGLE_180DEG: Angle = Angle(real::PI);
+    pub(crate) const ANGLE_360DEG: Angle = Angle(real::TAU);
+}
+
+/// Angle.
+///
+/// `Angle` is used to define the value of an angle.
+///
+/// # Examples
+///
+/// ## Create an `Angle` from a value
+///
+/// ```rust
+/// use embedded_graphics::geometry::Angle;
+///
+/// // Create an angle using the `from_degree` constructor method
+/// let angle_a = Angle::from_degree(10.0);
```

Comment:
Similar to my other comment, it would be useful to show `Angle::from_radian(s)` here as well.

Another example showing usage of the `AngleUnit` (with a doc link to it as well) would be handy too.